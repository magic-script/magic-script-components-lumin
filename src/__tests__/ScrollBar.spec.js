//
import React from "react";
import { create } from "react-test-renderer";
import { ScrollBar } from "magic-script-components";

describe("ScrollBar component", () => {
  test("Matches the snapshot", () => {
    const scrollbar = create(<ScrollBar width={0.1} />);
    expect(scrollbar.toJSON()).toMatchSnapshot();
  });
});
