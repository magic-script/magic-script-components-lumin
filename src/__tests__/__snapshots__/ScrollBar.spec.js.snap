// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ScrollBar component Matches the snapshot 1`] = `
<scrollBar
  width={0.1}
/>
`;
